app/ide-desktop/lib/dashboard/src/authentication/src/authentication/service.tsx

```diff
@@ -84,3 +137,56 @@ const loadAmplifyConfig = (
 
   return baseConfig as authConfig.AmplifyConfig;
 };
+
+const openUrlWithExternalBrowser = (url: string) => {
+  /** See {@link AuthenticationApi} for safety details. */
+  // @ts-expect-error
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  const authenticationApi: AuthenticationApi = window.authenticationApi;
```

Comment:
prefer module augmentation:

```ts
interface Window {
  authenticationApi: {};
}
```

(window is declared to be of type `Window` so augmenting `Window` will make the `window.authenticationApi` type-safe)

Reply:
Module has been augmented by adding this interface, works perfectly.